src/monitoring_service/cli.py

```diff
@@ -92,6 +92,12 @@ def validate_address(_ctx: Any, _param: Any, value: Optional[str]) -> Optional[s
     type=str,
     help='Path to SQLite3 db which stores the application state',
 )
+@click.option(
+    '--min-service-fee',
```

Comment:
I wonder if we should call this fee or reward...

Reply:
We are not quite consistent here all over the place. In the spec we call it both. Maybe define it once and for all? For the monitoring service I tend to "reward" or even "bounty"

Reply:
I will call it reward now

Reply:
`reward` and `min_reward` seems consistent and for the pathfinding service we will call it fee